pkg/kubernetes/node/v1alpha1/kubernetes_test.go

```diff
@@ -71,53 +86,80 @@ func TestWithDefaultOptions(t *testing.T) {
 
 func TestGetClientOrCached(t *testing.T) {
 	tests := map[string]struct {
-		expectErr  bool
-		KubeClient *Kubeclient
+		getClientSet        getClientsetFn
```

Comment:
can we add these as new test functions than modifying existing ones

Reply:
Done